### PR TITLE
AA KD, on /docs, in "Getting started", I see a tutorial on "Semantic segmentation"

### PR DESCRIPTION
**Pull request by FrankFacundo**
- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests

### CODE REVIEW

File: recipes/getting-started/getting_started-image_semantic_segmentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,824 @@
...
+   "metadata": {},
+   "source": [
+    "### How to label ?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Suppose that you've created a simple project as described in [Creating the project](#project). Now, to annotate, go to `Overview` and click on `Start Labeling`. You should arrive at the image of a car. Now just select a class (*Object A* or *Object B*) then you can drag the pointer around the object or make many straight lines until you finish to select an object.\n",
```
Comment (p-desaintchamas) [inline recipes/getting-started/getting_started-image_semantic_segmentation.ipynb:386]: Use correct names, not object A and Object B

**Review by p-desaintchamas — APPROVED** [review]

File: recipes/getting-started/getting_started-image_semantic_segmentation.ipynb (hunk 1 above)
Comment (FrankFacundo-KiliTechnology) [reply recipes/getting-started/getting_started-image_semantic_segmentation.ipynb]: I changed the class for Door

File: recipes/getting-started/getting_started-image_semantic_segmentation.ipynb [hunk 2]
```diff
@@ -0,0 +1,824 @@
...
+    "# What is semantic segmentation<a id='intro'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The task of image semantic segmentation consists on detecting specifig regions of objects in an image. In concrete terms, this means detecting the shape of objects of a different categories in images. Each category is composed of objects that have all common features, as an example, we can consider the task of image semantic segmentation for the categories _Circle_ and _Square_, we'll then try to identify these objects in our images knowing that circles are round and squares have four edges.\n",
+    "\n",
+    "In image semantic segmentation our goal is not only to know of the presence of given objects, but also to identify their positions, dimensions and shape. This require work at pixel level in order to classify every single pixel of the object. This can be achieved in an approximate way by drawing the shape of the object.\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-image_semantic_segmentation.ipynb:39]: This requires

File: recipes/getting-started/getting_started-image_semantic_segmentation.ipynb [hunk 3]
```diff
@@ -0,0 +1,824 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You arrive on a list of projects.\n",
+    "\n",
+    "![](../img/project_list.png)\n",
+    "\n",
+    "You can refer to this [document](https://cloud.kili-technology.com/docs/definitions/definitions/) to find the definitions of key concepts at Kili. One of them is a project : it is a combination of :\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-image_semantic_segmentation.ipynb:96]: In English, no space in punctuation before ` :`, ` !`, etc (check everywhere in the PR))

File: recipes/getting-started/getting_started-image_semantic_segmentation.ipynb [hunk 4]
```diff
@@ -0,0 +1,824 @@
...
+   "source": [
+    "Suppose that you've created a simple project as described in [Creating the project](#project). Now, to annotate, go to `Overview` and click on `Start Labeling`. You should arrive at the image of a car. Now just select a class, in this case we will use the class `Door` then you can drag the pointer around the object or make many straight lines until you finish to select an object.\n",
+    "\n",
+    "![](../img/getting_started/semantic-segmentation-annotation.gif)\n",
+    "\n",
+    "You can select the category you want by clicking on the right radio button, or by pressing the key underlined in the class name \"d\" for *Door*, \"w\" for *Wheel* and \"c\" for *Car*.\n",
+    "\n",
+    "Then, click on `Submit` to send the label.\n",
+    "\n",
+    "In addition, Kili have smart tools in order to make easier the annotation of semantic segmentation. You can use the [Interactive Segmentation](https://cloud.kili-technology.com/docs/image-interfaces/segmentation/#interactive-segmentation) model:\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-image_semantic_segmentation.ipynb:394]: Kili has

**Pull request merged**